configs/kde/root-image/etc/lightdm/Xsession

```diff
@@ -62,73 +62,50 @@ if [[ -z "$DBUS_SESSION_BUS_ADDRESS" ]]; then
-if [[ *'[]'* = "${_current_val}" ]]; then
-	gsettings set org.gnome.desktop.input-sources sources "[('xkb','us')]"
-fi
-
-# Enabled extensions
-_extensions="['user-theme@gnome-shell-extensions.gcampax.github.com', 'status-menu-buttons@dev.antergos.com', 'dash-to-dock@micxgx.gmail.com', 'panel-osd@berend.de.schouwer.gmail.com', 'topIcons@adel.gadllah@gmail.com']"
-gsettings set org.gnome.shell enabled-extensions "${_extensions}"
-
-# Extension - Panel-OSD Settings
-gsettings set org.gnome.shell.extensions.panel-osd x-pos 96.0
-gsettings set org.gnome.shell.extensions.panel-osd y-pos 96.0
-gsettings set org.gnome.shell.extensions.panel-osd force-expand true
-
-# Extension - dash-to-dock Settings
-gsettings set org.gnome.shell.extensions.dash-to-dock apply-custom-theme false
-gsettings set org.gnome.shell.extensions.dash-to-dock opaque-background true
-gsettings set org.gnome.shell.extensions.dash-to-dock background-opacity 0.5
-gsettings set org.gnome.shell.extensions.dash-to-dock custom-theme-running-dots false
-gsettings set org.gnome.shell.extensions.dash-to-dock custom-theme-shrink false
-
-# Randomly choose a default wallpaper from antergos-wallpapers
-DIR="/usr/share/antergos/wallpapers"
-PIC=$(ls ${DIR}/* | shuf -n1)
-
-gsettings set org.gnome.desktop.background picture-uri file://${PIC}
-
-# Set favorite apps
-gsettings set org.gnome.shell favorite-apps "['Cnchi.desktop', 'nautilus.desktop', 'chromium.desktop', 'pidgin.desktop', 'gnome-music.desktop']"
-
-# Disable screensaver
-gsettings set org.gnome.desktop.screensaver lock-enabled false
-gsettings set org.gnome.desktop.lockdown disable-lock-screen true
-gsettings set org.gnome.desktop.session idle-delay 0
-
-
-# Set theme
-gsettings set org.gnome.desktop.interface gtk-theme 'Numix-Frost-Light'
-gsettings set org.gnome.desktop.wm.preferences theme 'Numix-Frost-Light'
-gsettings set org.gnome.shell.extensions.user-theme name 'Numix-Frost'
-
-# Set icon theme
-gsettings set org.gnome.desktop.interface icon-theme 'Numix-Square'
-
-# Not show desktop icons
-gsettings set org.gnome.desktop.background show-desktop-icons false
-
-# Minimize and close buttons
-gsettings set org.gnome.desktop.wm.preferences button-layout 'appmenu:minimize,maximize,close'
-
-# Don't Disable terminal bell (accessibility concerns)
-gsettings set org.gnome.desktop.wm.preferences audible-bell true
-
-# Configure touchpad
-gsettings set org.gnome.desktop.peripherals.touchpad tap-to-click true
-gsettings set org.gnome.desktop.peripherals.touchpad natural-scroll true
-gsettings set org.gnome.desktop.peripherals.touchpad scroll-method 'two-finger-scrolling'
-
-# Set fonts
-gsettings set org.gnome.desktop.interface font-name 'Open Sans 12'
-gsettings set org.gnome.desktop.wm.preferences titlebar-font 'Open Sans 13'
-gsettings set org.gnome.settings-daemon.plugins.xsettings antialiasing 'rgba'
-
-# Turn on automatic date/time
-gsettings set org.gnome.desktop.datetime automatic-timezone true;
+	# Set KDE in .dmrc
```

Comment:
@Wyn10 I think that for KDE it isn't necessary to apply configuration here. The reason we do it here for GNOME is because using gsettings/dconf during the ISO build is unreliable at best (usually doesnt work at all). KDE is configured with config files in users' `/home` so we can apply those settings by placing necessary files in `/etc/skel` on the ISO.